src/configure/extract_from_c.ml

```diff
@@ -0,0 +1,118 @@
+(*
+ * Copyright (c) 2016 whitequark.
+ *
+ * This file is distributed under the terms of the MIT License.
+ * See the file LICENSE for details.
+ *)
+
+let getenv ~default name =
+  try Sys.getenv name
+  with Not_found -> default
+
+let nsplit sep str =
```

Comment:
I think that `nsplit` could be implemented like this:

```ocaml
Str.(split (regexp_string sep)) str
```

(The behaviour is slightly different when the separator appears at the beginning or the end of the string, but that probably doesn't matter.)